website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -1,107 +1,161 @@
 ---
-title: Installing the dbt Cloud CLI (Alpha)
+title: Installing the dbt Cloud CLI (Beta)
 id: cloud-cli-installation
 description: "Instructions for installing and configuring dbt Cloud CLI"
 ---
 
-:::warning Alpha functionality 
+:::warning Beta functionality 
 
-The following installation instructions are for the dbt Cloud CLI, currently in Alpha (actively in development and being tested). 
-
-These instructions are not intended for general audiences at this time. 
+The following installation instructions are for the dbt Cloud CLI, currently in Beta (actively in development).
 
 ::: 
 
-## Installing dbt Cloud CLI
+## About the Cloud CLI
+
+The Cloud CLI lets your run dbt commands against your dbt Cloud development environment from your local command line.
+
+dbt commands are run against dbt Cloud's infrastructure and benefit from:
+
+* Secure credential storage in the dbt Cloud platform.
+* Automatic deferral of build artifacts to your Cloud project's production environment. 
+* Speedier, lower cost builds.
+* Significant platform improvements, to be released over the coming months.
+
+After installing, you invoke the Cloud CLI the same way you would call dbt Core.  For example, to compile a project using dbt Cloud:
+
+```bash
+dbt deps
+dbt compile
+```
+
+
+## Installing the dbt Cloud CLI
+
+:::caution
+
+For compatibility, both the Cloud CLI and dbt Core are invoked by running `dbt`. This can cause a path conflict, where your OS invokes either the Cloud CLI or dbt Core depending on it's presence in your $PATH environment variable.
 
-### Install and update with Brew on MacOS (recommended)
+If you have dbt Core installed locally, ensure that you either deactivate your Python environment or uninstall before proceeding.  Alternatively, advanced users can modify the $PATH environment variable to correctly point to the dbt Cloud CLI binary to use both the Cloud CLI and dbt Core together.
 
-1. Install the dbt Cloud CLI: 
+You can always uninstall the Cloud CLI to return to using dbt Core.
+:::
+
+### Install and update with Brew - MacOS
+
```

Comment:
given homebrew isn’t going to be supported for core, is this still right? do we need to say anything about [bundles](https://github.com/dbt-labs/dbt-core-bundles)?